server/js/sockets.js

```diff
@@ -126,30 +128,30 @@ sockets.init = function socketInit(server) {
       const topicString = topic.split('/');
       if (topicString.length === 1) {
         switch (topic) {
-          case 'start':
+          case DAS.start:
             socket.emit('start');
             break;
-          case 'stop':
+          case DAS.stop:
             socket.emit('stop');
             break;
-          case 'data':
+          case DAS.data:
             mqttDataTopicHandler(socket, payloadString);
             if (sendToPublicMQTTBroker()) {
-              PUBLIC_MQTT_CLIENT.publish('data', payloadString);
+              PUBLIC_MQTT_CLIENT.publish(DAS.data, payloadString);
             }
             break;
           default:
             console.error(`Unhandled topic - ${topic}`);
             break;
         }
-      } else if (topicString[0] === 'power_model') {
+      } else if (topicString[0] === 'boost') {
         socket.emit('power-model-running');
         const message = mqttDataConverter(payloadString);
         switch (topicString[1]) {
           case 'predicted_max_speed':
             socket.emit('power-model-max-speed', message);
             break;
-          case 'recommended_SP':
+          case 'recommended_sp':
             socket.emit('power-model-recommended-SP', message);
             break;
           case 'plan_generated':
```

Comment:
If we want to generalise this as well, we could use an object which wraps the topic (e.g. `BOOST.predicted_max_speed.name` instead of 'predicted_max_speed'). Or maybe some sort of topic parser similar to how `topicString` is derived which we can use to index/access different parts of the topic.